Upstream: no.
Reason: Without it the tool is linked staticlly.
    GH has an issue requesting an option to build against the shared lib.
URL: https://github.com/google/brotli/issues/803

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,7 +183,7 @@
 
 # Build the brotli executable
 add_executable(brotli ${BROTLI_CLI_C})
-target_link_libraries(brotli ${BROTLI_LIBRARIES_STATIC})
+target_link_libraries(brotli ${BROTLI_LIBRARIES})
 
 # Installation
 if(NOT BROTLI_BUNDLED_MODE)
